src/styles.css

```diff
@@ -1281,3 +1281,29 @@ a {
 th {
   font-weight: normal;
 }
+
+.loading::after {
+  display: inline-block;
+  animation: dotty steps(1, end) 1s infinite;
+  content: "";
+  min-width: 0.6rem;
+  text-align: left;
+}
+
+@keyframes dotty {
+  0% {
+    content: "";
+  }
+  25% {
+    content: ".";
+  }
+  50% {
+    content: "..";
+  }
+  75% {
+    content: "...";
+  }
+  100% {
+    content: "";
+  }
+}
```

Comment:
I would like some feedback on this loader, perhaps a better way to do this as well.

Reply:
I faced some issues with the CSS approach to this when changing fonts.

We've got the `<Loading />` component which does this programatically using JS.